src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -94,7 +96,7 @@ func logPanic(msg string) {
 		panic("Error initializing logging")
 	}
 
-	<-logging.Configure("", "", cfg.InstanceId, version, revisionDate)
+	<-logging.Configure("", "", "", version, revisionDate)
```

Comment:
I figured it wasn't a big deal for a panic, but I just added it anyway since I changed the settings.Load call to be in init (so it now has it when it gets here). Thanks @fffw good catch.